ui/src/views/NotFoundPage/NotFoundPage.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <cv-content id="#main-content">
+    <div class="page__details">
+      <h2 class="aside__header">Page Not Found</h2>
+    </div>
+    <div class="bx--grid bx--grid--full-width landing-page">
+      <div class="bx--row landing-page__r2">
+        <div class="bx--col-lg-16">
+          <section>
+            <aside class="content--register">
```

Comment:
Could you add a aria-label and translatable strings here like:
```vue
            <aside class="content--register" :aria-label="$t('redirecting')">
              <p>{{ $t('redirecting') }}</p>
            </aside>
```

And then add the "redirecting ..." string to `src/plugins/en.json`